src/Hook/ProductSearch.php

```diff
@@ -50,9 +50,16 @@ public function productSearchProvider(array $params)
         // Query is an instance of:
         // PrestaShop\PrestaShop\Core\Product\Search\ProductSearchQuery
         if ($query->getIdCategory()) {
-            $this->context->controller->addJS($this->module->getPathUri() . 'views/dist/front.js');
-            $this->context->controller->addCSS($this->module->getPathUri() . 'views/dist/front.css');
-            $this->context->controller->addJS($this->module->getPathUri() . 'views/dist/cldr.js');
```

Comment:
no more CLDR in front office ?

Reply:
Yes it is: https://github.com/PrestaShop/ps_facetedsearch/blob/master/_dev/front/slider.js#L27
This file was removed few weeks ago.